src/sections/Partners/Partners.module.scss

```diff
@@ -0,0 +1,68 @@
+@import "../../styles/vars.scss";
+
+.section {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  max-width: 640px;
+
+  a {
+    text-decoration: none;
+  }
+}
+
+.partnersIconsWrapper {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap:20px;
+}
+
+.partnerLink {
+  flex-basis: 150px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+
+  &:hover {
```

Comment:
I'm not sure about the behaviour on hover for the logos. If the logos would be graphics, the wouldn't affect them. Maybe we could make them slightly bigger on hover? Smth like

https://user-images.githubusercontent.com/20808724/146237756-4314c9cb-a582-4b53-b2bd-b17d01bce222.mov

 wdyt?